### PR TITLE
sink: trivial fix for mq sink

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve? 
None

### What is changed and how it works?
1. rename SendMessage to AsyncSendMessage
2. add comments

### Check List 

Tests 

 - No code

Code changes

None

Side effects

None

Related changes

None

### Release note 

```release-note
None
```

### CODE REVIEW

**Comment by hi-rustin** [comment]
/cc @ben1009 @liuzix 

Could you please take a look? Thanks!

File: cdc/sink/mq.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func newMqSink(
 		return nil, errors.Trace(err)
 	}
 
-	k := &mqSink{
+	s := &mqSink{
```
Comment (3AceShowHand) [inline cdc/sink/mq.go:102]: why `k` to `s` ?

Comment (hi-rustin) [reply cdc/sink/mq.go:102]: Because this is a Sink not a Kakfa 😆

**Review by lance6716 — APPROVED** [review]

**Comment by ben1009** [comment]
just rename + comments ?

**Comment by hi-rustin** [comment]
> just rename + comments ?

Yes.

**Review by ben1009 — APPROVED** [review]

**Comment by hi-rustin** [comment]
/assgin @liuzix 

Could you please take a look and help me trigger the merge process? Thanks!

**Pull request merged**